kotest-property/src/jvmMain/kotlin/io/kotest/property/arbitrary/dates.kt

```diff
@@ -85,20 +82,46 @@ fun Arb.Companion.localTime(): Arb<LocalTime> = arbitrary(listOf(LocalTime.of(23
  * @see [localTime]
  */
 fun Arb.Companion.localDateTime(
-   minYear: Int = 1970,
-   maxYear: Int = 2030
+   minYear: Int,
+   maxYear: Int
+): Arb<LocalDateTime> {
```

Comment:
@sksamuel this is an API breaking change, right?. What should be done here?

Any usage of:
```kotlin
Arb.localDateTime(minYear = 0, maxYear = 3000)
```

Reply:
Okay, I see another way of fixing the overload resolution ambiguity I came across earlier. I'll make some changes and request your code review again. Thanks!

Reply:
The code is great, we just want to avoid breaking changes.
Anyone doing `Arb.localDateTime(minYear = 0)` will now have a compile error, same as `Arb.localDateTime(maxYear = 3000)`

I think simple way around that is two more overloads, each just expecting a single argument.